code/obj/item/misc_weapons.dm

```diff
@@ -1717,6 +1717,122 @@ obj/item/whetstone
 		throwforce = 20 //higher base damage, lower once the slasher starts scaling up their machete
 		force = 20
 
+
+// Halberd- Experimental weapon by NightmareChamillian
+/obj/item/halberd
+	name = "Halberd"
+	desc = "An ancient axe-like weapon capable of cleaving and piercing flesh with ease. You have no idea what this is doing outside a museum."
+	icon = 'icons/obj/large/64x32.dmi'
+	icon_state = "halberd"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	item_state = "halberd1"
+
+	w_class = W_CLASS_BULKY
+	two_handed = 1
+	throw_range = 10
+	throwforce = 30 //yeet like spear
+	stamina_crit_chance = 5
+
+	//these combat variables change depending on intent- starts with help intent vars
+	force = 28
+	stamina_damage = 10
+	stamina_cost = 7
+
+
+	hit_type = DAMAGE_CUT
+	flags = FPRINT | TABLEPASS | USEDELAY | ONBACK
+	c_flags = EQUIPPED_WHILE_HELD
+	item_function_flags = USE_INTENT_SWITCH_TRIGGER | USE_SPECIALS_ON_ALL_INTENTS
+
+	var/guard = null
```

Comment:
Add a code doc comment (`///` on line above or `//!` on same line) explaining what this var is for